app/api/v2/handlers/agent_api.py

```diff
@@ -26,9 +26,13 @@ def add_routes(self, app: web.Application):
         router.add_get('/deploy_commands', self.get_deploy_commands)
         router.add_get('/deploy_commands/{ability_id}', self.get_deploy_commands_for_ability)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'],
+                          summary="Retrieve all agents",
+                          description="Retrieve all agents by criteria. Use fields from the "
+                                      "'AgentSchema' in the request body to filter the retrieved agents.")
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
-    @aiohttp_apispec.response_schema(AgentSchema(many=True, partial=True))
+    @aiohttp_apispec.response_schema(AgentSchema(many=True, partial=True),
+                          description="Returns a list of all agents in 'AgentSchema' format.")
```

Comment:
I think flake8 may complain about `description` being under-indented here

Reply:
Roger, fixing in next commit